fbxosctrl.py

```diff
@@ -476,8 +562,11 @@ def __init__(self, controller):
         self.cmdCallbacks = {
             'regapp': self.controller.registerApp,
             'wifistatus': self.controller.getWifiStatus,
-            'wifion': self.controller.setWifiOn,
-            'wifioff': self.controller.setWifiOff,
+            'woff': self.controller.setWifiOn,
+            'won': self.controller.setWifiOff,
+            'wifiplan': self.controller.getWifiPlanning,
```

Comment:
I suggest 'wpstatus' instead of 'wifiplan': this is more explicit about what the command does.
In the same manner, you could modify 'wifistatus' to use 'wrstatus' to have same pattern.